src/http/websocket_http_client.zig

```diff
@@ -86,22 +89,25 @@ fn buildRequestBody(
         host_.deinit();
     }
 
+    const host_fmt = ZigURL.HostFormatter{
+        .is_https = is_https,
+        .host = host_.slice(),
```

Comment:
I would change this and line 110 to use `byteSlice()` to handle the utf16 case to be safe